src/vocation.h

```diff
@@ -86,6 +86,10 @@ class Vocation
 			return fromVocation;
 		}
 
+		bool isProtected() const {
+			return _protected;
```

Comment:
please don't use variables starting with underscore

Reply:
Not possible in this scenario

Reply:
why?

Reply:
It's an access modifier keyword like `public` & `private`